ci/jenkins/jobs/projects-lab.yaml

```diff
@@ -1518,4 +1574,11 @@
                 fail: true
                 timeout: 135
                 type: absolute
-            publishers: []
+            publishers: 
+            - archive:
+              allow-empty: true
+              artifacts: antrea-test-logs.tar.gz
```

Comment:
Name the log file with job name

Reply:
Renaming the file will cause the CI packaging command to fail. https://github.com/antrea-io/antrea/blob/b5bfc5eb527c389afbdde5078b685c707a4129c5/ci/jenkins/test.sh#L669

Reply:
I meant include job name in logfile name e.g antrea-kind-e2e-logs, it will be hard to identify the log corresponding to job, multiple jobs are having same logfile name

Reply:
Although the log file names are the same, they are saved under different Jenkins job paths, this will not cause conflicts when saving files. Using unique file names would require modifying the corresponding CI scripts, and since it would affect almost all jobs, it's better to discuss it in a new issue. But it should not block the current PR. 

I have a concern in only one case, @KMAnju-2021 do you know if we allow two same kind jobs(e.g. two e2e jobs) to run simultaneously on one testbed? If so, we need to ensure that the log files are saved in different directories in such cases.

Reply:
Since each job runs in its own workspace, even if multiple jobs run simultaneously, they will be saved under different paths !

Reply:
Yes, there should be no conflict issues on Jenkins; otherwise, we would already be facing issues with the existing jobs.